internal/docker/builder.go

```diff
@@ -336,9 +333,7 @@ func (d *Builder) construct(bprint b.Blueprint) (errs []error) {
 			Author:    "Complement",
 			Pause:     true,
 			Reference: "localhost/complement:" + res.contextStr,
-			Config: &container.Config{
-				Labels: labels,
-			},
+			Changes:   changes,
```

Comment:
I'd prefer it if we kept the map of `labels` and then just called `toChanges(labels) []string` or something.